doc/mgr/dashboard.rst

```diff
@@ -333,6 +333,21 @@ commands::
 
   $ ceph dashboard ac-user-create <username> <password> administrator
 
+Enable a Locked User
+^^^^^^^^^^^^^^^^^^^^
+
+If the user is locked after multiple invalid login attempt, the user needs to
```

Comment:
If a user account is disabled as a result of multiple invalid login attempts, then it needs to be manually enabled by the administrator. This can be done by the following command::

Reply:
Done.